javaslang-benchmark/src/test/java/javaslang/benchmark/BenchmarkPerformanceReporter.java

```diff
@@ -0,0 +1,366 @@
+package javaslang.benchmark;
+
+import javaslang.Tuple2;
+import javaslang.collection.Array;
+import javaslang.collection.CharSeq;
+import javaslang.collection.List;
+import javaslang.collection.Map;
+import javaslang.collection.TreeMap;
+import javaslang.control.Option;
+import org.openjdk.jmh.infra.BenchmarkParams;
+import org.openjdk.jmh.results.BenchmarkResult;
+import org.openjdk.jmh.results.Result;
+import org.openjdk.jmh.results.RunResult;
+
+import java.text.DecimalFormat;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.stream.Collectors;
+
+public class BenchmarkPerformanceReporter {
```

Comment:
Will do! :)

I actually started working on this class well before I saw your aggregator, and then by the time I added the aggregators you didn't support groups.